[2025-07-02 17:35:01,114: INFO: main: logging is implemeted]
[2025-07-03 11:58:10,436: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-07-03 11:58:10,440: INFO: common: yaml file: params.yaml loaded successfully]
[2025-07-03 11:58:10,445: INFO: common: created directory at: artifacts]
[2025-07-03 11:58:10,448: INFO: common: created directory at: artifacts/data_ingestion]
[2025-07-03 11:58:11,122: INFO: 111464816: Downloaded file and saved to artifacts/data_ingestion/dataset.zip]
[2025-07-03 12:00:38,571: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-07-03 12:00:38,573: INFO: common: yaml file: params.yaml loaded successfully]
[2025-07-03 12:00:38,575: INFO: common: created directory at: artifacts]
[2025-07-03 12:00:38,581: INFO: common: created directory at: artifacts/data_ingestion]
[2025-07-03 12:00:42,265: INFO: 2917796623: Downloaded file and saved to artifacts/data_ingestion/dataset.zip]
[2025-07-03 12:00:42,404: INFO: 2917796623: Extracted zip file to artifacts/data_ingestion]
[2025-07-03 12:01:10,595: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-07-03 12:01:10,597: INFO: common: yaml file: params.yaml loaded successfully]
[2025-07-03 12:01:10,598: INFO: common: created directory at: artifacts]
[2025-07-03 12:01:10,599: INFO: common: created directory at: artifacts/data_ingestion]
[2025-07-03 12:01:15,257: INFO: 2917796623: Downloaded file and saved to artifacts/data_ingestion/dataset.zip]
[2025-07-03 12:01:15,386: INFO: 2917796623: Extracted zip file to artifacts/data_ingestion]
[2025-07-03 12:14:11,503: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-07-03 12:14:11,505: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-07-03 12:14:11,505: INFO: common: yaml file: params.yaml loaded successfully]
[2025-07-03 12:14:11,506: INFO: common: created directory at: artifacts]
[2025-07-03 12:14:11,506: INFO: common: created directory at: artifacts/data_ingestion]
[2025-07-03 12:14:11,508: INFO: data_ingestion: File already exists ]
[2025-07-03 12:14:11,642: INFO: data_ingestion: Extracted zip file to artifacts/data_ingestion]
[2025-07-03 12:14:11,642: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<]
[2025-07-03 12:19:42,174: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-07-03 12:19:42,174: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-07-03 12:19:42,174: INFO: common: yaml file: params.yaml loaded successfully]
[2025-07-03 12:19:42,174: INFO: common: created directory at: artifacts]
[2025-07-03 12:19:42,174: INFO: common: created directory at: artifacts/data_ingestion]
[2025-07-03 12:19:50,491: INFO: data_ingestion: Downloaded file and saved to artifacts/data_ingestion/dataset.zip]
[2025-07-03 12:19:50,630: INFO: data_ingestion: Extracted zip file to artifacts/data_ingestion]
[2025-07-03 12:19:50,630: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<]
[2025-07-03 12:53:32,948: INFO: config: PyTorch version 2.7.1 available.]
[2025-07-03 13:04:52,175: INFO: config: PyTorch version 2.7.1 available.]
[2025-07-03 13:04:52,819: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-07-03 13:04:52,827: INFO: common: yaml file: params.yaml loaded successfully]
[2025-07-03 13:04:52,830: INFO: common: created directory at: artifacts]
[2025-07-03 13:04:52,835: INFO: common: created directory at: artifacts/data_transformation]
[2025-07-03 13:08:03,602: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-07-03 13:08:03,614: INFO: common: yaml file: params.yaml loaded successfully]
[2025-07-03 13:08:03,614: INFO: common: created directory at: artifacts]
[2025-07-03 13:08:03,620: INFO: common: created directory at: artifacts/data_transformation]
[2025-07-03 13:17:28,539: INFO: config: PyTorch version 2.7.1 available.]
[2025-07-03 13:17:29,199: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-07-03 13:17:29,205: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-07-03 13:17:29,210: INFO: common: yaml file: params.yaml loaded successfully]
[2025-07-03 13:17:29,212: INFO: common: created directory at: artifacts]
[2025-07-03 13:17:29,212: INFO: common: created directory at: artifacts/data_ingestion]
[2025-07-03 13:17:29,212: INFO: data_ingestion: File already exists ]
[2025-07-03 13:17:29,412: INFO: data_ingestion: Extracted zip file to artifacts/data_ingestion]
[2025-07-03 13:17:29,412: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<]
[2025-07-03 13:17:29,412: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2025-07-03 13:17:29,412: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-07-03 13:17:29,412: INFO: common: yaml file: params.yaml loaded successfully]
[2025-07-03 13:17:29,418: INFO: common: created directory at: artifacts]
[2025-07-03 13:17:29,419: INFO: common: created directory at: artifacts/data_transformation]
[2025-07-03 13:17:37,300: INFO: main: >>>>>> stage Data Transformation stage completed <<<<<<]
[2025-07-03 13:46:54,218: INFO: config: PyTorch version 2.7.1 available.]
[2025-07-03 13:56:21,967: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-07-03 13:56:22,013: INFO: common: yaml file: params.yaml loaded successfully]
[2025-07-03 13:56:22,017: INFO: common: created directory at: artifacts]
[2025-07-03 13:56:22,022: INFO: common: created directory at: artifacts/model_trainer]
[2025-07-03 13:56:27,024: WARNING: file_download: Xet Storage is enabled for this repo, but the 'hf_xet' package is not installed. Falling back to regular HTTP download. For better performance, install the package with: `pip install huggingface_hub[hf_xet]` or `pip install hf_xet`]
[2025-07-03 13:58:27,895: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-07-03 13:58:27,923: INFO: common: yaml file: params.yaml loaded successfully]
[2025-07-03 13:58:27,932: INFO: common: created directory at: artifacts]
[2025-07-03 13:58:27,935: INFO: common: created directory at: artifacts/model_trainer]
[2025-07-03 13:58:38,095: WARNING: file_download: Xet Storage is enabled for this repo, but the 'hf_xet' package is not installed. Falling back to regular HTTP download. For better performance, install the package with: `pip install huggingface_hub[hf_xet]` or `pip install hf_xet`]
[2025-07-03 14:01:18,655: INFO: config: PyTorch version 2.7.1 available.]
[2025-07-03 14:01:22,757: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-07-03 14:01:22,771: INFO: common: yaml file: params.yaml loaded successfully]
[2025-07-03 14:01:22,776: INFO: common: created directory at: artifacts]
[2025-07-03 14:01:22,782: INFO: common: created directory at: artifacts/model_trainer]
[2025-07-03 14:01:36,708: WARNING: file_download: Xet Storage is enabled for this repo, but the 'hf_xet' package is not installed. Falling back to regular HTTP download. For better performance, install the package with: `pip install huggingface_hub[hf_xet]` or `pip install hf_xet`]
[2025-07-03 15:18:25,472: INFO: config: PyTorch version 2.7.1 available.]
[2025-07-03 15:19:09,105: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-07-03 15:19:09,123: INFO: common: yaml file: params.yaml loaded successfully]
[2025-07-03 15:19:09,134: INFO: common: created directory at: artifacts]
[2025-07-03 15:19:09,137: INFO: common: created directory at: artifacts/model_evaluation]
[2025-07-03 18:35:47,164: INFO: config: PyTorch version 2.7.1 available.]
[2025-07-03 18:35:51,834: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-07-03 18:35:51,845: INFO: common: yaml file: params.yaml loaded successfully]
[2025-07-03 18:35:51,848: INFO: common: created directory at: artifacts]
[2025-07-03 18:35:51,849: INFO: common: created directory at: artifacts/model_evaluation]
[2025-07-03 18:44:08,029: INFO: rouge_scorer: Using default tokenizer.]
[2025-07-03 18:56:00,140: INFO: config: PyTorch version 2.7.1 available.]
[2025-07-03 18:56:01,644: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-07-03 18:56:01,649: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-07-03 18:56:01,660: INFO: common: yaml file: params.yaml loaded successfully]
[2025-07-03 18:56:01,660: INFO: common: created directory at: artifacts]
[2025-07-03 18:56:01,660: INFO: common: created directory at: artifacts/data_ingestion]
[2025-07-03 18:56:01,660: INFO: data_ingestion: File already exists ]
[2025-07-03 18:56:01,855: INFO: data_ingestion: Extracted zip file to artifacts/data_ingestion]
[2025-07-03 18:56:01,855: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<]
[2025-07-03 18:56:01,855: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2025-07-03 18:56:01,864: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-07-03 18:56:01,867: INFO: common: yaml file: params.yaml loaded successfully]
[2025-07-03 18:56:01,869: INFO: common: created directory at: artifacts]
[2025-07-03 18:56:01,869: INFO: common: created directory at: artifacts/data_transformation]
[2025-07-03 18:56:04,966: ERROR: main: [Errno 22] Invalid argument: 'C:/Users/Welcome/Desktop/TextSummarizer/artifacts/data_transformation/samsum_dataset/train/data-00000-of-00001.arrow']
Traceback (most recent call last):
  File "C:\Users\Welcome\Desktop\TextSummarizer\main.py", line 20, in <module>
    data_transformation.initiate_data_transformation()
  File "C:\Users\Welcome\Desktop\TextSummarizer\src\text_summarizer\pipeline\stage2_data_transformation.py", line 13, in initiate_data_transformation
    data_transformation.convert()
  File "C:\Users\Welcome\Desktop\TextSummarizer\src\text_summarizer\components\data_transformation.py", line 29, in convert
    dataset_samsum_pt.save_to_disk(os.path.join(self.config.root_dir, "samsum_dataset"))
  File "C:\Users\Welcome\Desktop\TextSummarizer\venv\lib\site-packages\datasets\dataset_dict.py", line 1348, in save_to_disk
    dataset.save_to_disk(
  File "C:\Users\Welcome\Desktop\TextSummarizer\venv\lib\site-packages\datasets\arrow_dataset.py", line 1557, in save_to_disk
    for job_id, done, content in Dataset._save_to_disk_single(**kwargs):
  File "C:\Users\Welcome\Desktop\TextSummarizer\venv\lib\site-packages\datasets\arrow_dataset.py", line 1581, in _save_to_disk_single
    writer = ArrowWriter(
  File "C:\Users\Welcome\Desktop\TextSummarizer\venv\lib\site-packages\datasets\arrow_writer.py", line 388, in __init__
    self.stream = self._fs.open(path, "wb")
  File "C:\Users\Welcome\Desktop\TextSummarizer\venv\lib\site-packages\fsspec\spec.py", line 1310, in open
    f = self._open(
  File "C:\Users\Welcome\Desktop\TextSummarizer\venv\lib\site-packages\fsspec\implementations\local.py", line 201, in _open
    return LocalFileOpener(path, mode, fs=self, **kwargs)
  File "C:\Users\Welcome\Desktop\TextSummarizer\venv\lib\site-packages\fsspec\implementations\local.py", line 365, in __init__
    self._open()
  File "C:\Users\Welcome\Desktop\TextSummarizer\venv\lib\site-packages\fsspec\implementations\local.py", line 370, in _open
    self.f = open(self.path, mode=self.mode)
OSError: [Errno 22] Invalid argument: 'C:/Users/Welcome/Desktop/TextSummarizer/artifacts/data_transformation/samsum_dataset/train/data-00000-of-00001.arrow'
[2025-07-03 19:03:59,509: INFO: config: PyTorch version 2.7.1 available.]
[2025-07-03 19:04:00,551: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-07-03 19:04:00,551: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-07-03 19:04:00,559: INFO: common: yaml file: params.yaml loaded successfully]
[2025-07-03 19:04:00,562: INFO: common: created directory at: artifacts]
[2025-07-03 19:04:00,562: INFO: common: created directory at: artifacts/data_ingestion]
[2025-07-03 19:04:00,562: INFO: data_ingestion: File already exists ]
[2025-07-03 19:04:00,707: INFO: data_ingestion: Extracted zip file to artifacts/data_ingestion]
[2025-07-03 19:04:00,708: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<]
[2025-07-03 19:04:00,709: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2025-07-03 19:04:00,713: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-07-03 19:04:00,713: INFO: common: yaml file: params.yaml loaded successfully]
[2025-07-03 19:04:00,718: INFO: common: created directory at: artifacts]
[2025-07-03 19:04:00,718: INFO: common: created directory at: artifacts/data_transformation]
[2025-07-03 19:04:03,561: ERROR: main: [Errno 22] Invalid argument: 'C:/Users/Welcome/Desktop/TextSummarizer/artifacts/data_transformation/samsum_dataset/train/data-00000-of-00001.arrow']
Traceback (most recent call last):
  File "C:\Users\Welcome\Desktop\TextSummarizer\main.py", line 20, in <module>
    data_transformation.initiate_data_transformation()
  File "C:\Users\Welcome\Desktop\TextSummarizer\src\text_summarizer\pipeline\stage2_data_transformation.py", line 13, in initiate_data_transformation
    data_transformation.convert()
  File "C:\Users\Welcome\Desktop\TextSummarizer\src\text_summarizer\components\data_transformation.py", line 29, in convert
    dataset_samsum_pt.save_to_disk(os.path.join(self.config.root_dir, "samsum_dataset"))
  File "C:\Users\Welcome\Desktop\TextSummarizer\venv\lib\site-packages\datasets\dataset_dict.py", line 1348, in save_to_disk
    dataset.save_to_disk(
  File "C:\Users\Welcome\Desktop\TextSummarizer\venv\lib\site-packages\datasets\arrow_dataset.py", line 1557, in save_to_disk
    for job_id, done, content in Dataset._save_to_disk_single(**kwargs):
  File "C:\Users\Welcome\Desktop\TextSummarizer\venv\lib\site-packages\datasets\arrow_dataset.py", line 1581, in _save_to_disk_single
    writer = ArrowWriter(
  File "C:\Users\Welcome\Desktop\TextSummarizer\venv\lib\site-packages\datasets\arrow_writer.py", line 388, in __init__
    self.stream = self._fs.open(path, "wb")
  File "C:\Users\Welcome\Desktop\TextSummarizer\venv\lib\site-packages\fsspec\spec.py", line 1310, in open
    f = self._open(
  File "C:\Users\Welcome\Desktop\TextSummarizer\venv\lib\site-packages\fsspec\implementations\local.py", line 201, in _open
    return LocalFileOpener(path, mode, fs=self, **kwargs)
  File "C:\Users\Welcome\Desktop\TextSummarizer\venv\lib\site-packages\fsspec\implementations\local.py", line 365, in __init__
    self._open()
  File "C:\Users\Welcome\Desktop\TextSummarizer\venv\lib\site-packages\fsspec\implementations\local.py", line 370, in _open
    self.f = open(self.path, mode=self.mode)
OSError: [Errno 22] Invalid argument: 'C:/Users/Welcome/Desktop/TextSummarizer/artifacts/data_transformation/samsum_dataset/train/data-00000-of-00001.arrow'
[2025-07-03 19:05:42,031: INFO: config: PyTorch version 2.7.1 available.]
[2025-07-03 19:05:43,214: INFO: main: >>>>>> stage Data Ingestion stage started <<<<<<]
[2025-07-03 19:05:43,222: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-07-03 19:05:43,231: INFO: common: yaml file: params.yaml loaded successfully]
[2025-07-03 19:05:43,231: INFO: common: created directory at: artifacts]
[2025-07-03 19:05:43,231: INFO: common: created directory at: artifacts/data_ingestion]
[2025-07-03 19:05:43,231: INFO: data_ingestion: File already exists ]
[2025-07-03 19:05:43,412: INFO: data_ingestion: Extracted zip file to artifacts/data_ingestion]
[2025-07-03 19:05:43,418: INFO: main: >>>>>> stage Data Ingestion stage completed <<<<<<]
[2025-07-03 19:05:43,418: INFO: main: >>>>>> stage Data Transformation stage started <<<<<<]
[2025-07-03 19:05:43,420: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-07-03 19:05:43,420: INFO: common: yaml file: params.yaml loaded successfully]
[2025-07-03 19:05:43,420: INFO: common: created directory at: artifacts]
[2025-07-03 19:05:43,420: INFO: common: created directory at: artifacts/data_transformation]
[2025-07-03 19:05:45,723: ERROR: main: [Errno 22] Invalid argument: 'C:/Users/Welcome/Desktop/TextSummarizer/artifacts/data_transformation/samsum_dataset/train/data-00000-of-00001.arrow']
Traceback (most recent call last):
  File "C:\Users\Welcome\Desktop\TextSummarizer\main.py", line 20, in <module>
    data_transformation.initiate_data_transformation()
  File "C:\Users\Welcome\Desktop\TextSummarizer\src\text_summarizer\pipeline\stage2_data_transformation.py", line 13, in initiate_data_transformation
    data_transformation.convert()
  File "C:\Users\Welcome\Desktop\TextSummarizer\src\text_summarizer\components\data_transformation.py", line 31, in convert
    dataset_samsum_pt.save_to_disk(str(output_path))
  File "C:\Users\Welcome\Desktop\TextSummarizer\venv\lib\site-packages\datasets\dataset_dict.py", line 1348, in save_to_disk
    dataset.save_to_disk(
  File "C:\Users\Welcome\Desktop\TextSummarizer\venv\lib\site-packages\datasets\arrow_dataset.py", line 1557, in save_to_disk
    for job_id, done, content in Dataset._save_to_disk_single(**kwargs):
  File "C:\Users\Welcome\Desktop\TextSummarizer\venv\lib\site-packages\datasets\arrow_dataset.py", line 1581, in _save_to_disk_single
    writer = ArrowWriter(
  File "C:\Users\Welcome\Desktop\TextSummarizer\venv\lib\site-packages\datasets\arrow_writer.py", line 388, in __init__
    self.stream = self._fs.open(path, "wb")
  File "C:\Users\Welcome\Desktop\TextSummarizer\venv\lib\site-packages\fsspec\spec.py", line 1310, in open
    f = self._open(
  File "C:\Users\Welcome\Desktop\TextSummarizer\venv\lib\site-packages\fsspec\implementations\local.py", line 201, in _open
    return LocalFileOpener(path, mode, fs=self, **kwargs)
  File "C:\Users\Welcome\Desktop\TextSummarizer\venv\lib\site-packages\fsspec\implementations\local.py", line 365, in __init__
    self._open()
  File "C:\Users\Welcome\Desktop\TextSummarizer\venv\lib\site-packages\fsspec\implementations\local.py", line 370, in _open
    self.f = open(self.path, mode=self.mode)
OSError: [Errno 22] Invalid argument: 'C:/Users/Welcome/Desktop/TextSummarizer/artifacts/data_transformation/samsum_dataset/train/data-00000-of-00001.arrow'
[2025-07-03 19:49:05,999: INFO: server: Started server process [2116]]
[2025-07-03 19:49:06,006: INFO: on: Waiting for application startup.]
[2025-07-03 19:49:06,007: INFO: on: Application startup complete.]
[2025-07-03 19:49:06,008: INFO: server: Uvicorn running on http://0.0.0.0:8000 (Press CTRL+C to quit)]
[2025-07-03 19:51:10,348: INFO: server: Shutting down]
[2025-07-03 19:51:10,472: INFO: on: Waiting for application shutdown.]
[2025-07-03 19:51:10,472: INFO: on: Application shutdown complete.]
[2025-07-03 19:51:10,472: INFO: server: Finished server process [2116]]
[2025-07-03 19:51:45,532: INFO: server: Started server process [23492]]
[2025-07-03 19:51:45,536: INFO: on: Waiting for application startup.]
[2025-07-03 19:51:45,536: INFO: on: Application startup complete.]
[2025-07-03 19:51:45,543: INFO: server: Uvicorn running on http://0.0.0.0:8000 (Press CTRL+C to quit)]
[2025-07-03 19:52:52,908: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-07-03 19:52:52,923: INFO: common: yaml file: params.yaml loaded successfully]
[2025-07-03 19:52:52,923: INFO: common: created directory at: artifacts]
[2025-07-03 19:52:52,923: INFO: common: created directory at: artifacts/model_evaluation]
[2025-07-03 19:55:11,518: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-07-03 19:55:11,541: INFO: common: yaml file: params.yaml loaded successfully]
[2025-07-03 19:55:11,546: INFO: common: created directory at: artifacts]
[2025-07-03 19:55:11,548: INFO: common: created directory at: artifacts/model_evaluation]
[2025-07-03 20:00:22,290: INFO: server: Shutting down]
[2025-07-03 20:00:22,420: INFO: on: Waiting for application shutdown.]
[2025-07-03 20:00:22,442: INFO: on: Application shutdown complete.]
[2025-07-03 20:00:22,443: INFO: server: Finished server process [23492]]
[2025-07-03 20:03:28,186: INFO: server: Started server process [13308]]
[2025-07-03 20:03:28,186: INFO: on: Waiting for application startup.]
[2025-07-03 20:03:28,186: INFO: on: Application startup complete.]
[2025-07-03 20:03:28,186: INFO: server: Uvicorn running on http://0.0.0.0:8000 (Press CTRL+C to quit)]
[2025-07-03 20:10:42,196: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-07-03 20:10:42,207: INFO: common: yaml file: params.yaml loaded successfully]
[2025-07-03 20:10:42,209: INFO: common: created directory at: artifacts]
[2025-07-03 20:10:42,211: INFO: common: created directory at: artifacts/model_evaluation]
[2025-07-03 20:11:27,781: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-07-03 20:11:27,788: INFO: common: yaml file: params.yaml loaded successfully]
[2025-07-03 20:11:27,790: INFO: common: created directory at: artifacts]
[2025-07-03 20:11:27,792: INFO: common: created directory at: artifacts/model_evaluation]
[2025-07-03 20:13:56,031: INFO: server: Shutting down]
[2025-07-03 20:13:56,161: INFO: on: Waiting for application shutdown.]
[2025-07-03 20:13:56,179: INFO: on: Application shutdown complete.]
[2025-07-03 20:13:56,181: INFO: server: Finished server process [13308]]
[2025-07-03 20:16:25,231: INFO: server: Started server process [22196]]
[2025-07-03 20:16:25,233: INFO: on: Waiting for application startup.]
[2025-07-03 20:16:25,240: INFO: on: Application startup complete.]
[2025-07-03 20:16:25,242: INFO: server: Uvicorn running on http://0.0.0.0:8000 (Press CTRL+C to quit)]
[2025-07-03 20:17:44,721: INFO: common: yaml file: config\config.yaml loaded successfully]
[2025-07-03 20:17:44,729: INFO: common: yaml file: params.yaml loaded successfully]
[2025-07-03 20:17:44,731: INFO: common: created directory at: artifacts]
[2025-07-03 20:17:44,731: INFO: common: created directory at: artifacts/model_evaluation]
[2025-07-03 20:18:34,250: INFO: server: Shutting down]
[2025-07-03 20:18:34,382: INFO: on: Waiting for application shutdown.]
[2025-07-03 20:18:34,382: INFO: on: Application shutdown complete.]
[2025-07-03 20:18:34,382: INFO: server: Finished server process [22196]]
